packages/web-config-server/src/apiV1/measureBuilders/checkMultiConditionsByOrgUnit.js

```diff
@@ -0,0 +1,91 @@
+import { DataBuilder } from '/apiV1/dataBuilders/DataBuilder';
+import { checkValueSatisfiesCondition } from '@tupaia/utils';
+import { groupAnalyticsByOrgUnit } from './helpers';
+
+/**
+ * Example use case: If yes to SchCVD012 but no to SchCVD012a display 'Yes', if yes to SchCVD012 and yes to SchCVD012a display 'Yes, smart TV'
+ * Example orgUnitConditions config:
+ * orgUnitConditions: {
+    Yes: {
+      SchCVD012: {
+        value: 'Yes',
+        operator: '=',
+      },
```

Comment:
Let's make this comment more concise (so that it doesn't add much clutter to the file). An example:

```suggestion
    Yes: {
      SchCVD012: { value: 'Yes', operator: '=' },
```

Also, the `No` condition can be removed (two conditions, perhaps even one should be enough to illustrate their usage)

Finally, super detail but since we're updating that we could also add a ` * ` in the start of every line, to maintain a DocBlock structure (this detail could potentially matter if we were to extract doco from code)